Sources/XCTest/XCTAssert.swift

```diff
@@ -100,15 +100,15 @@ private func _XCTEvaluateAssertion(assertion: _XCTAssertion, @autoclosure messag
     }
 }
 
-/// This function emits a test failure if the general Bool expression passed
-/// to it evaluates to false.
+/// This function emits a test failure if the general `Boolean` expression passed
+/// to it evaluates to `false`.
```

Comment:
Ah, I see. I haven't been aware of that name change.
So, I'll keep the changes, as they were (if that's the consensus).